static/app/views/auth/login.spec.tsx

```diff
@@ -55,7 +55,7 @@ describe('Login', function () {
 
     render(<Login {...routerProps} />);
 
-    expect(await screen.findByRole('link', {name: 'Register'})).toBeInTheDocument();
+    expect(await screen.findByText('Register')).toBeInTheDocument();
```

Comment:
the tab component uses `role=“tab”`, so we can do:

```suggestion
    expect(await screen.findByRole('tab', {name: 'Register'})).toBeInTheDocument();
```